packages/@frograming/frogger/src/models/Board.js

```diff
@@ -94,6 +94,11 @@ export default class Board {
     }));
   }
 
+  isWaterUpOfFrog () {
```

Comment:
Should this be `isRiverUpOfFrog`?

Reply:
So far `isXXUp` means `XX` is directly of the frog. So I think `isRiverUpOfFrog` would be a confusing name. Since we are checking `3 <= frog.pos.y <= 7`, would it be nicer to name it `isFrogInRiver` or something else instead? and should we provide the same for the road?

Reply:
Renaming to isRiverUpOfFrog.

Is does implement is river up, since the river is rows 2 to 6. So if you are on row 7, you aren't in the river, but the river is "up" of you. At row 2, you are in the river, but isRiverUpOfFrog returns false.